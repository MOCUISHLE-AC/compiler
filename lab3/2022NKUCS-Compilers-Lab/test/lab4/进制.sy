int main(){
    //进制部分测试
    int BIN=65535;
    int OCT=0177777;
    int Hex=0xffff;
	return 0;
}