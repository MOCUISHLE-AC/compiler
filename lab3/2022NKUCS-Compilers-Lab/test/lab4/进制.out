[DEBUG LAB4]: 	INT	int
[DEBUG LAB4]: 	MAIN	main
[DEBUG LAB4]: 	LPAREN	(
[DEBUG LAB4]: 	RPAREN	)
[DEBUG LAB4]: 	LBRACE	{
[DEBUG LAB4]: 	行注释://进制部分测试
[DEBUG LAB4]: 	INT	int
[DEBUG LAB4]: 	ID:	BIN
[DEBUG LAB4]: 	ASSIGN	=
[DEBUG LAB4]: 	DECIMIAL:	65536
[DEBUG LAB4]: 	SEMICOLON	;
[DEBUG LAB4]: 	INT	int
[DEBUG LAB4]: 	ID:	OCT
[DEBUG LAB4]: 	ASSIGN	=
[DEBUG LAB4]: 	OCTONARY:	65535
[DEBUG LAB4]: 	SEMICOLON	;
[DEBUG LAB4]: 	INT	int
[DEBUG LAB4]: 	ID:	Hex
[DEBUG LAB4]: 	ASSIGN	=
[DEBUG LAB4]: 	HEXADECIMAL:	65535
[DEBUG LAB4]: 	SEMICOLON	;
[DEBUG LAB4]: 	RETURN	return
[DEBUG LAB4]: 	DECIMIAL:	0
[DEBUG LAB4]: 	SEMICOLON	;
[DEBUG LAB4]: 	RBRACE	}
